californium-core/pom.xml

```diff
@@ -37,6 +37,35 @@
 	<build>
 		<plugins>
 			<plugin>
+				<groupId>org.apache.felix</groupId>
+				<artifactId>maven-bundle-plugin</artifactId>
+				<extensions>true</extensions>
+				<configuration>
+					<instructions>
+						<Export-Package>
+							org.eclipse.californium.core,
```

Comment:
OK, probably that is the reason for the OSGi-ified versions in Eclipse Orbit. But I think Californium can not rely on these artifacts. At least I am not sure if they are provided in a Maven Repository.